tools/system_libs.py

```diff
@@ -894,6 +896,9 @@ def get_files(self):
     network_dir = shared.path_from_root('system', 'lib', 'libc', 'musl', 'src', 'network')
     return [os.path.join(network_dir, x) for x in LIBC_SOCKETS]
 
+  def can_use(self):
+    return super(libsockets, self).can_use() and not settings.PROXY_POSIX_SOCKETS
```

Comment:
is this right? I think the one after it is, but am not sure here.

Reply:
Yes, I believe the `libsockets` and `libsockets_proxy` provide the same API so they cannot be included at the same time without generating duplicate symbol errors.  

You can see here are they are used (under normal circumstances) in a mutually exclusive manor:
https://github.com/emscripten-core/emscripten/blob/d35de0bf3572dfe9759ac3c7e56b9ce5983f68ac/tools/system_libs.py#L1621-L1624

Reply:
Thanks, now I see. lgtm!